java-sdk/src/main/resources/akka.conf

```diff
@@ -15,31 +15,31 @@ akka {
   }
 
 
-//  ssl-config {
-//    trustManager = {
-//      stores = [
-//        {
-//          type = "PEM"
-//          path = ${?HARNESS_CA_CERT}
-//        }
-//      ]
-//    }
-//  }
-//
-//  ssl-config.enabledCipherSuites = [
-//    "TLS_DHE_RSA_WITH_AES_128_GCM_SHA256",
-//    "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
-//    "TLS_DHE_RSA_WITH_AES_256_GCM_SHA384",
-//    "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
-//  ]
-//
-//  ssl-config.debug = {
-//    ssl = true
-//    handshake = true
-//    verbose = true
-//    data = true
-//    trustmanager = true
-//    keymanager = true
-//  }
+  ssl-config {
+    trustManager = {
+      stores = [
+        {
+          type = "PEM"
+          path = ${?HARNESS_CA_CERT}
+        }
+      ]
+    }
+  }
+
+  ssl-config.enabledCipherSuites = [
+    "TLS_DHE_RSA_WITH_AES_128_GCM_SHA256",
+    "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
+    "TLS_DHE_RSA_WITH_AES_256_GCM_SHA384",
+    "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
+  ]
+
+  ssl-config.debug = {
+    ssl = true
+    handshake = true
+    verbose = true
+    data = true
+    trustmanager = true
+    keymanager = true
+  }
 
```

Comment:
I tried commenting this out and leaving it in for the sdk, but neither works with the server

Reply:
@pferrel  can I merge feature/security_defaults here?